src/yardstick/tool/grype.py

```diff
@@ -25,12 +32,19 @@ def __init__(
         path: str,
         version_detail: Optional[str] = None,
         env: Optional[Dict[str, str]] = None,
+        profile: Optional[GrypeProfile] = None,
         **kwargs,  # pylint: disable=unused-argument
     ):
+        if not profile:
+            profile = GrypeProfile()
+        self.profile = profile
```

Comment:
nit: should all the `self.x` be defined in the same new line block?

I don't know if there is a linter rule for this and apologies for the nit comment

```
if not profile:
    profile = GrypeProfile()

self.profile = profile
self.path = path
self._env = env
```